policyengine_us/variables/gov/ed/pell_grant/pell_grant.py

```diff
@@ -28,6 +28,6 @@ def formula(person, period, parameters):
                 eligibility == eligibility.possible_values.MAXIMUM,
                 eligibility == eligibility.possible_values.MINIMUM,
             ],
-            [0, p.amount.max, amount],
+            [0, min_(coa, p.amount.max), amount],
```

Comment:
could amount ever exceed coa?

capping as the very final step could be safer if so